viz-lib/src/visualizations/chart/getOptions.ts

```diff
@@ -23,6 +23,10 @@ const DEFAULT_OPTIONS = {
   // dateTimeFormat: 'DD/MM/YYYY HH:mm', // will be set from visualizationsSettings
   textFormat: "", // default: combination of {{ @@yPercent }} ({{ @@y }} ± {{ @@yError }})
 
+  enableLink: true,
```

Comment:
Let's make this `false` by default

Reply:
`enableLink` still appears to be true. This option should not be enabled for new graphs by default